open-hackathon-client/src/client/static/js/manage.controller.js

```diff
@@ -345,7 +345,7 @@ angular.module('oh.controllers', [])
       });
     };
 
-    var refresh = function() {
+    function refresh() {
```

Comment:
`function refresh() {...}` is exactly identical with `var refresh = function() {...}` except the latter is more OO-look